helper_test.go

```diff
@@ -751,6 +753,167 @@ func createRunTask(t *testing.T, client *Client, org *Organization) (*RunTask, f
 	}
 }
 
+func createRegistryProvider(t *testing.T, client *Client, org *Organization, registryName RegistryName) (*RegistryProvider, func()) {
+	var orgCleanup func()
+
+	if org == nil {
+		org, orgCleanup = createOrganization(t, client)
+	}
+
+	if (registryName != PublicRegistry) && (registryName != PrivateRegistry) {
+		t.Fatal("RegistryName must be public or private")
+	}
+
+	ctx := context.Background()
+
+	namespaceName := "test-namespace-" + randomString(t)
+	if registryName == PrivateRegistry {
+		namespaceName = org.Name
+	}
+
+	options := RegistryProviderCreateOptions{
+		Name:         "test-registry-provider-" + randomString(t),
+		Namespace:    namespaceName,
+		RegistryName: registryName,
+	}
+
+	prv, err := client.RegistryProviders.Create(ctx, org.Name, options)
+
+	if err != nil {
+		t.Fatal(err)
```

Comment:
Following up @brandonc re this [comment](https://github.com/hashicorp/go-tfe/pull/405/files#r876368479), I think we're OK using Fatal per my [comment](https://github.com/hashicorp/go-tfe/pull/405/files#r879789207).